### ISSUE TITLE
Suggestion: tab/shift-tab as ergonomic alias for alt+right/left arrow

### ISSUE THREAD

**Issue by aral** [helix-editor/helix#2561]
The select previous and next adjacent sibling key bindings are very useful for quickly navigating your code.

It feels like <kbd>tab</kbd> (for next adjacent sibling) and <kbd>shift</kbd><kbd>tab</kbd> for (previous adjacent sibling) would make ergonomic and semantically intuitive aliases for <kbd>alt</kbd><kbd>→</kbd> and <kbd>alt</kbd><kbd>←</kbd>.

**Comment by the-mikedavis** [comment]
Yeah, `A-poin` is setup mainly to use `A-p` as previous and `A-n` as next sibling which are very unambiguous and probably easier to remember. `A-o` for expand_selection and `A-i` for shrink_selection are a little less clear but kinda mimic the jumplist.

The point in not using `hjkl` or arrow keys for the bindings is that using `A-l`/`A-Right` can move you down in the document if you're up high in the tree where nodes are split across lines, like in:

```
function_call(
  some_long_arg1,
  some_long_arg2,
  some_long_arg3
)
```

For example if you had `some_long_arg2` selected and hit `A-l` or `A-Right`, you would most likely (depending on how the syntax tree gets parsed) end up on `some_long_arg3` which is downwards in the document.

Elsewhere in the keymap, `l` always moves you right (window management keys for example). So it can be confusing that `l` may move you down in the document in some cases.

See also https://github.com/helix-editor/helix/issues/2202#issuecomment-1104567608
